styles/base.css

```diff
@@ -4,62 +4,58 @@
   font-weight: normal;
   font-style: normal;
 }
-body {
+@font-face {
+  font-family: "Montserrat", sans-serif;
+  src: url("./Montserrat/Montserrat-Bold.ttf") format("truetype");
+  font-weight: 900;
+}
+@font-face {
+  font-family: "Montserrat", sans-serif;
+  src: url("./Montserrat/Montserrat-Light.ttf") format("truetype");
+  font-weight: 100;
+}
+@font-face {
+  font-family: "Montserrat", sans-serif;
+  src: url("./Montserrat/Montserrat-Regular.ttf") format("truetype");
+  font-weight: 500;
+}
+.body {
   font-family: "Montserrat", sans-serif;
   scroll-behavior: smooth;
 }
 .container {
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  box-sizing: border-box;
-  padding-left: 75px;
-  padding-right: 75px;
-  margin-left: auto;
-  margin-right: auto;
   max-width: 1150px;
+  padding: 10px 20px;
```

Comment:
Tutaj jeszcze zabrakło `margin: 0 auto`, aby wyśrodkować zawartość. Na mniejszym ekranie tego nie widać, ale jak masz większy to lepiej, aby odstępy pojawiły się po bokach niż całość była przykjelona do lewego boku.